OrbitGl/TimeGraphLayout.h

```diff
@@ -13,6 +13,7 @@ class TimeGraphLayout {
   float GetTextCoresHeight() const { return m_CoresHeight * scale_; }
   float GetEventTrackHeight() const { return m_EventTrackHeight * scale_; }
   float GetGraphTrackHeight() const { return m_GraphTrackHeight * scale_; }
+  float GetGraphTrackBoxWidth() const { return graph_track_box_width_; }
```

Comment:
nit. I would rename to GetGraphTrackValueLabelWidth() or something similar. I think "BoxWidth" might be too generic.

Reply:
Thanks!